lib/storage/src/dispatcher.rs

```diff
@@ -118,6 +118,18 @@ impl Dispatcher {
                             }
                         }
                     }
+
+                    if let Some(uuid) = &op.create_collection.uuid {
+                        log::warn!(
+                            "Collection UUID {uuid} explicitly specified, \
+                             when proposing create collection {} operation, \
+                             new random UUID will be generated instead",
+                            op.collection_name,
+                        );
+                    }
```

Comment:
We have to include UUID into Qdrant's `CreateCollection` structure, because the same type is used both in consensus (to be serialized to CBOR and written into WAL) and in REST API, so I've excluded UUID from OpenAPI spec and added this warning.

Reply:
Do we strictly have to regenerate here? Can't we just use what is provided, if we ever want to explicitly use this (as hidden API)?

Reply:
Not really, I just felt it's safer this way, but sure, we can allow it.